src/components/form-editor/form-editor.component.tsx

```diff
@@ -163,6 +168,7 @@ const FormEditor: React.FC = () => {
                       schema={schema}
                       onSchemaChange={updateSchema}
                       isLoading={isLoadingFormOrSchema}
+                      validateStateSetter={validationSetter}
```

Comment:
```suggestion
                      validateStateSetter={setIsValidating}
```

Reply:
Actually, I would rename `validateStateSetter` to `setIsValidating`

Reply:
The issues cited have been resolved, kindly review @samuelmale